utils/SnapPy/Snappy/BombIsotopeFractions.py

```diff
@@ -0,0 +1,76 @@
+import os
+import csv
+import re
+
+class BombIsotopeFractions:
+    # fractions{isotope} = {0: frac0, 10: frac10, 20: frac20} with 0-40 in hours and frac0-frac40 fractions of total fission-products in Bq
+    _fractions = None
+    _timesteps = [0,10,20,30,40]
+    def __new__(cls):
```

Comment:
Standard python textbook singleton pattern. Why would one use a lru-cache of size 1 instead of the singleton pattern directly? Sounds like cannons on sparrows. (I would like to see a `@singleton` in standard-python, though, no matter how it is implemented)

Reply:
In python 3.8 and later we have [`cached_property`](https://docs.python.org/3/library/functools.html#functools.cached_property)